internals/deps/index.js

```diff
@@ -36,7 +36,11 @@ function getDependencies() {
     return;
   }
 
-  return Promise.all(packageConfig.getPackageDependencies().map(dep => get({ source: dep.name })));
+  return Promise.all(
```

Comment:
We use prettier as our code formatter. Just ran it for this snippet, its supposed to be a one liner :) You can quick-fix it with the `lint:fix` script

Reply:
I don't know why the push checks didn't detect that. I'll investigate that.